packages/react/src/primitives/Button/__tests__/Button.test.tsx

```diff
@@ -46,6 +54,94 @@ describe('Button test suite', () => {
     expect(menu.classList).toContain(`${ComponentClassNames['Button']}--menu`);
   });
 
+  it.each(SUPPORTED_COLOR_THEMES)(
+    'should render the %s color theme for the default variation',
+    async (colorTheme) => {
+      const testId = `default-${colorTheme}-ColorTheme`;
+      render(
+        <Button
+          testId={testId}
+          colorTheme={colorTheme as ButtonColorTheme}
+        ></Button>
+      );
+      const button = await screen.findByTestId(testId);
+      expect(button.classList).toContain(
+        `amplify-button--outlined--${colorTheme}`
+      );
+    }
+  );
+
+  it.each(SUPPORTED_COLOR_THEMES)(
+    'should render the %s color theme for the primary variation',
+    async (colorTheme) => {
+      const testId = `primary-${colorTheme}-ColorTheme`;
+      render(
+        <Button
+          variation="primary"
+          testId={testId}
+          colorTheme={colorTheme as ButtonColorTheme}
+        ></Button>
```

Comment:
NIt: prefer self inclosing tags for JSX
```suggestion
        />
```